android-todo-list/architecture-samples-main/app/src/main/java/com/example/android/architecture/blueprints/todoapp/util/StatsIgUtil.kt

```diff
@@ -43,4 +50,26 @@ object StatsIgUtil {
         return Statsig.checkGate(FEATURE_GATE_DELETE_TODO_ACCESS)
     }
 
+    /**
+     * The function to register the event without metadata in the Statsig portal.
+     *
+     * @param eventName Name of the event to be logged.
+     */
+    fun eventLogWithoutMetadata(eventName: String) {
+        Statsig.logEvent(eventName)
+    }
+
+    /**
+     * The function to register the event with metadata in the Statsig portal.
+     *
+     * @param eventName Name of the event to be logged.
+     */
+    fun eventLogWithMetadata(
```

Comment:
in general, the parameters to your function are documentation enough, adding a `withMetadata` is unnecessary.

But again, just delete these functions

Reply:
Will remove it.